src/wp-admin/about.php

```diff
@@ -49,214 +49,398 @@
-				</h3>
+				<div class="about__image">
+					<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false">
+						<rect width="48" height="48" rx="4" fill="#1E1E1E"/>
+						<path d="M23.8 15.38C23.2 15.38 22.685 15.165 22.255 14.735C21.825 14.305 21.61 13.79 21.61 13.19C21.61 12.59 21.825 12.075 22.255 11.645C22.685 11.215 23.2 11 23.8 11C24.4 11 24.915 11.215 25.345 11.645C25.775 12.075 25.99 12.59 25.99 13.19C25.99 13.79 25.775 14.305 25.345 14.735C24.915 15.165 24.4 15.38 23.8 15.38ZM20.59 35V19.01C19.23 18.91 17.915 18.77 16.645 18.59C15.375 18.41 14.16 18.18 13 17.9L13.45 16.1C15.15 16.5 16.84 16.785 18.52 16.955C20.2 17.125 21.96 17.21 23.8 17.21C25.64 17.21 27.4 17.125 29.08 16.955C30.76 16.785 32.45 16.5 34.15 16.1L34.6 17.9C33.44 18.18 32.225 18.41 30.955 18.59C29.685 18.77 28.37 18.91 27.01 19.01V35H25.21V27.2H22.39V35H20.59Z" fill="white"/>
+					</svg>
+				</div>
+				<h3 class="is-smaller-heading"><?php _e( 'Improved admin and editor accessibility' ); ?></h3>
 				<p>
-					<?php _e( 'Now you can lock your blocks. Choose to disable the option to move a block, remove a block, or both. This simplifies project handover, allowing your clients to unleash their creativity without worrying about accidentally breaking their site in the process.' ); ?>
+					<?php
+					printf(
+						__( 'More than 40 improvements in accessibility include resolving focus loss problems in the editor, improving form labels and audible messages, making alternative text easier to edit, and fixing the sub-menu overlap in the expanded admin side navigation at smaller screen sizes and higher zoom levels. Learn more about <a href="%s">accessibility in WordPress</a>.' ),
+						'https://wordpress.org/about/accessibility/'
+					);
+					?>
 				</p>
 			</div>
+			<div class="column">
+				<div class="about__image">
+					<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg" aria-hidden="true" focusable="false">
+						<rect width="48" height="48" rx="4" fill="#1E1E1E"/>
+						<path fill-rule="evenodd" clip-rule="evenodd" d="M23.2639 25.5896L18.7107 30.2783L17.6346 29.2333L22.1471 24.5865L16.0056 24.5598L16.0121 23.0598L22.2329 23.0869L17.9693 18.6964L19.0454 17.6514L23.2449 21.9759L23.2136 16.0062L24.7136 15.9983L24.7446 21.9117L28.8818 17.6514L29.9579 18.6964L25.6798 23.1019L32.1563 23.13L32.1498 24.63L25.7955 24.6024L30.2926 29.2333L29.2165 30.2783L24.7644 25.6937L24.7983 32.1497L23.2983 32.1576L23.2639 25.5896Z" fill="white"/>
+					</svg>
+				</div>
+				<h3 class="is-smaller-heading"><?php _e( 'Other notes of interest' ); ?></h3>
+				<p><?php _e( '6.1 includes a new time-to-read feature showing content authors the approximate time-to-read values for pages, posts, and custom post types.' ); ?></p>
+				<p><?php _e( 'The site tagline is empty by default in new sites but can be modified in General Settings.' ); ?></p>
```

Comment:
When I suggested mentioning the site tagline, I wanted a link to the "General Settings" page (if the user has the proper permissions).

```
<?php 
	printf(
		/* translators: %s: "General Settings" admin page title, linked to the page if the user can edit options. */
		__( 'The site tagline is empty by default in new sites but can be modified in %s.' ),
		current_user_can( 'manage_options' ) ? '<a href="' . esc_url( admin_url( 'options-general.php' ) ) . '">' . __( 'General Settings' ) . '</a>' : __( 'General Settings' )
	);
?>
```

Reply:
Added in https://github.com/WordPress/wordpress-develop/pull/3433/commits/b2d1dbc7082a2950abd7caac51f97b5f22a88496 👍🏻